src/game/eop.js

```diff
@@ -116,321 +109,324 @@ export const ElevationOfPrivilege = Game({
     }
     return ret;
   },
-  moves: {
-    toggleModal: (G, ctx) => {
-      // if the player has passed, they shouldn't be able to toggle the modal
-      let passed = [...G.passed];
-      if (passed.includes(ctx.playerID) || (G.threat.modal && G.threat.owner !== ctx.playerID)) {
-        return INVALID_MOVE;
-      }
-      let card = getDealtCard(G);
-      return {
-        ...G,
-        threat: {
-          ...G.threat,
-          modal: !G.threat.modal,
-          new: true,
-          owner: ctx.playerID,
-          type: G.suit,
-          id: uuidv4(),
-          title: "",
-          severity: "Medium",
-          description: getThreatDescription(card),
-          mitigation: "",
-        },
-      }
-    },
-    toggleModalUpdate: (G, ctx, threat) => {
-      // if the player has passed, they shouldn't be able to toggle the modal
-      let passed = [...G.passed];
-      if (passed.includes(ctx.playerID) || (threat.owner !== ctx.playerID)) {
-        return INVALID_MOVE;
-      }
 
-      return {
-        ...G,
-        threat: {
-          ...G.threat,
-          modal: !G.threat.modal,
-          new: false,
-          id: threat.id,
-          owner: ctx.playerID,
-          title: threat.title,
-          type: threat.type,
-          severity: threat.severity,
-          description: threat.description,
-          mitigation: threat.mitigation,
-        },
-      }
-    },
-    updateThreat: (G, ctx, field, value) => {
-      return {
-        ...G,
-        threat: {
-          ...G.threat,
-          [field]: value,
+  playerView: PlayerView.STRIP_SECRETS,
+  endIf: (G, ctx) => {
+    if (G.round > G.maxRounds) {
+      let scores = [...G.scores];
+      let winner = 0, max = -1;
+      for (let i=0; i<scores.length; i++) {
```

Comment:
Would this be better readable?
```js
return scores.indexOf(
  Math.max(scores)
);
```

Reply:
agreed, I've kept the variable winner around just to make it obvious what the function returns but it's not needed. Let me know if you think I should remove it

```javascript
export function endGameIf(G, ctx) {
  if (G.round > G.maxRounds) {
    const scores = [...G.scores];
    const winner = scores.indexOf(Math.max(scores));
    return winner;
  }
}
```

Reply:
I like it